setup.py

```diff
@@ -60,22 +60,35 @@ def run(self):
     name            = name,
     version         = version_ns['__version__'],
     packages        = packages,
-    description     = "Jupyter protocol implementation and client libraries",
+    description     = 'Jupyter protocol implementation and client libraries',
     author          = 'Jupyter Development Team',
     author_email    = 'jupyter@googlegroups.com',
     url             = 'https://jupyter.org',
     license         = 'BSD',
     platforms       = "Linux, Mac OS X, Windows",
     keywords        = ['Interactive', 'Interpreter', 'Shell', 'Web'],
+    project_urls    = {
+        'Documentation': 'https://jupyter-client.readthedocs.io',
+        'Source': 'https://github.com/jupyter/jupyter_client/',
+        'Tracker': 'https://github.com/jupyter/jupyter_client/issues',
+    },
     classifiers     = [
+        'Framework :: Jupyter',
         'Intended Audience :: Developers',
+        'Intended Audience :: Education',
         'Intended Audience :: System Administrators',
         'Intended Audience :: Science/Research',
         'License :: OSI Approved :: BSD License',
+        'Operating System :: MacOS',
+        'Operating System :: Microsoft :: Windows',
+        'Operating System :: POSIX :: Linux',
```

Comment:
I'd expect it to work on any platform where Python and ZMQ are available, which is probably a much bigger group than these three. Maybe use `Operating System :: OS Independent` instead? Or leave it out altogether - I think OS-independent is the default assumption for Python modules.

Reply:
Sure.